eta/core/data.py

```diff
@@ -531,6 +532,78 @@ def remove_schema(self):
         '''Removes the enforced schema from the container.'''
         self.schema = None
 
+    def has_attr_with_name(self, name):
+        '''Returns whether or not the container contains an Attribute with the
+        give name.
+
+        Args:
+            name: the Attribute name
+
+        Returns:
+            boolean indicating whether or not the container contains an
+                Attribute with the given name
+        '''
+        return bool(self.get_attrs_with_name(name))
```

Comment:
It would be more efficient for large containers to do:
```py
for attr in self:
    if attr.name == name:
        return True

return False
```

Reply:
I got this one, since I added this method.

Reply:
Fixed.